io/retry/retry.go

```diff
@@ -42,7 +43,11 @@ func main() {
 
 		if err := command.Run(); err != nil {
 			if !isRetriable(tool, err) {
-				os.Exit(err.(*exec.ExitError).Sys().(syscall.WaitStatus).ExitStatus())
+				if err, ok := err.(*exec.ExitError); ok {
+					os.Exit(err.Sys().(syscall.WaitStatus).ExitStatus())
+				}
+				fmt.Fprintf(os.Stderr, "Failed to run command: %v\n", err)
```

Comment:
Just log.Fatalf?

Reply:
That leaves a bunch of extra goo at the front (timestamp, filenames etc)